java-squid/src/main/java/org/sonar/java/se/symbolicvalues/RelationState.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * SonarQube Java
+ * Copyright (C) 2012-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.java.se.symbolicvalues;
+
+import org.sonar.java.se.ConstraintManager.BooleanConstraint;
+
+public enum RelationState {
+  Fulfilled {
+    @Override
+    public boolean rejects(BooleanConstraint constraint) {
+      return BooleanConstraint.FALSE.equals(constraint);
+    }
+
+    @Override
+    public boolean isDetermined() {
+      return true;
+    }
+  },
+  UnFulfilled {
+    @Override
+    public boolean rejects(BooleanConstraint constraint) {
+      return BooleanConstraint.TRUE.equals(constraint);
+    }
+
+    @Override
+    public boolean isDetermined() {
+      return true;
+    }
+  },
+  Undetermined {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS00115)